httplib2shim/__init__.py

```diff
@@ -114,7 +125,13 @@ def _conn_request(self, conn, request_uri, method, body, headers):
             scheme, host, conn.port, request_uri)
 
         decode = True if method != 'HEAD' else False
-
+        if isinstance(self.pool, urllib3.ProxyManager):
```

Comment:
no, urllib3 did parse the proxy_url, but it didn't create or update proxy_headers. however, I already simplify the code by create proxy_headers beforehand, and remove that if-clause in my last commit.